python/ray/autoscaler/_private/event_summarizer.py

```diff
@@ -59,6 +59,8 @@ def summary(self) -> List[str]:
         with self.lock:
             out = []
             for template, quantity in self.events_by_key.items():
+                # Indicate explicitly this is a summarized message over time.
+                template = "[SUMMARY] " + template
```

Comment:
huh ideally we should summarize all the nodes with different types? i.e. instead of
```
            "[SUMMARY] Adding 1 node(s) of type ray.worker.new.",
            "[SUMMARY] Adding 1 node(s) of type ray.worker.old.",
```

we should show

```
[SUMMARY] 10 nodes(s) has been added...
[SUMMARY] 5 nodes(s) has been removed..
```

Reply:
yeah, this would probabbly mean we passing different message templates in. 

But I guess not summarizing different types is also okay?

Reply:
e.g. 
```
[SUMMARY] 5 nodes(s) of type <> has been removed.
```

Reply:
yeah, can we do that instead?